Build fails because of browserslist warning

Spent quite a bit of time trying to build an old project with errors due to a third-party library:

```
WARN  meteorApp:  STDERR: Browserslist: caniuse-lite is outdated. Please run next command `npm update caniuse-lite browserslist`

ERROR  meteorApp:  build was terminated by meteor-desktop as some errors were reported to stderr, you should see it above, also check meteor.log for more info, to ignore it use the --ignore-stderr "<string>"
npm ERR! code ELIFECYCLE
npm ERR! errno 1
```

For some reason the `--ignore-stderr` flag did not work.

https://github.com/wojtkowiak/meteor-desktop/blob/3e4f21be1b7ca3cfbf9093b413037e679e502d81/lib/meteorApp.js#L474

--- a/lib/meteorApp.js
+++ b/lib/meteorApp.js
@@ -474,6 +474,7 @@ export default class MeteorApp {
                     !~line.indexOf('--production') &&
                     !~line.indexOf('Output exceeds ') &&
                     !~line.indexOf('Node#moveTo') &&
+                    !~line.indexOf('Browserslist') && 
                     (
                         Array.isArray(self.$.env.options.ignoreStderr) &&
                         self.$.env.options.ignoreStderr.every(str => !~line.indexOf(str))

